src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -499,20 +499,20 @@ class DevTask extends AbstractFeatureTask {
         @Override
         public boolean updateArtifactPaths(ProjectModule projectModule, boolean redeployCheck, boolean generateFeatures, ThreadPoolExecutor executor)
                 throws PluginExecutionException {
-            // unable to identify the changes made, showing option for user. always return false because action is invoked manually
+            // for multi module, unable to identify the changes made, showing option for user. return true to trigger recompile
             if (isMultiModuleProject()) {
-                info("A change was detected in a build file. The libertyDev task could not determine if a server restart is required.");
-                info("To restart the server, type 'r' and press Enter.");
+                warn("A change was detected in a build file. The libertyDev task could not determine if a server restart is required.");
```

Comment:
Why did you take out the message about how to restart the server?

Reply:
As discussed earlier, there is a chance that due to the length of recompileJava build messages  our restart message may get written away.
What I understood from the previous comment was just to show the warning message. so the "Press r to Restart" part was removed

Reply:
Let's combine it into one long warning message then. We will get feedback at playback on changes after that.